pkg/autoscaler/config/config.go

```diff
@@ -45,11 +45,12 @@ func defaultConfig() *autoscalerconfig.Config {
 		ContainerConcurrencyTargetFraction: defaultTargetUtilization,
 		ContainerConcurrencyTargetDefault:  100,
 		// TODO(#1956): Tune target usage based on empirical data.
-		TargetUtilization:             defaultTargetUtilization,
-		RPSTargetDefault:              200,
-		MaxScaleUpRate:                1000,
-		MaxScaleDownRate:              2,
-		TargetBurstCapacity:           200,
+		TargetUtilization: defaultTargetUtilization,
```

Comment:
That was done by gofmt...  think the alignment blocks are delineated by comments (which is why L48-51 got realigned). If I move the comment to the end of the line (instead of on a new line before) the indentation remains the same.